envelope.md

```diff
@@ -55,6 +52,29 @@ envelopes with individual signatures.
 }
 ```
 
+### No signatures
+
+An envelope MAY have zero signatures to indicate unsigned data. For example,
```

Comment:
I need to think about this one a bit. It feels like a potential alg=none style footgun. Using a separate data structure/serialization for unsigned data feels safer IMO.

Reply:
Know what, I'm going to revert that particular one and send a separate PR for it.

Reply:
Sent out #38 to discuss. Let's not block this PR on that.

Reply:
Yes, what Dan says makes sense. Even for an initially unsigned payload that needs to be signed in turn by multiple people, someone can start by signing it first, turning it into a signed payload, then pass it around to attach more signatures.